LazyCache/CachingService.cs

```diff
@@ -34,6 +37,17 @@ public CachingService(Func<ICacheProvider> cacheProviderFactory)
 
         }
 
+        [ActivatorUtilitiesConstructor]
+        public CachingService(ICacheProvider cacheProvider, IOptions<LazyCacheOptions> options)
+        {
+            if (cacheProvider == null) throw new ArgumentNullException(nameof(cacheProvider));
+            _options = options.Value;
+            this.cacheProvider = new Lazy<ICacheProvider>(() => cacheProvider);
+            keyLocks = new int[_options.NumberOfKeyLocks];
+            DefaultCachePolicy.DefaultCacheDurationSeconds = _options.DefaultCacheDurationSeconds;
+

```

Comment:
Minor Suggestion: Consider removing this redundant empty line for cleaner code presentation.